index.js

```diff
@@ -288,17 +314,14 @@ converters.ImportSpecifier = (path, context) => {
       };
     }
 
-    return convert(exported, {
-      ...context,
```

Comment:
Why can't you use object spread?

Reply:
Tests were failing on using spread. Not sure of why.